ChangeLog.md

```diff
@@ -1,6 +1,15 @@
 # DGtal 0.9
 
 ## New Features / Critical Changes
+- *Geometry Package*
+
+ - New Alpha Thick Segment Computer allowing the recognition of noisy
+   digital segment (from a given thickness parameter). The actual
```

Comment:
"of noisy digital segment" -> "of thick digital segment, adapted to noisy contours" ?